libs/components/packages/package.json

```diff
@@ -87,7 +88,8 @@
       "ag-grid-angular": "^32.2.2",
       "ag-grid-community": "^32.2.2",
       "ag-grid-enterprise": "^32.1.0",
-      "autonumeric": "^4.10.5"
+      "autonumeric": "^4.10.5",
+      "skyux-eslint": "0.0.0-PLACEHOLDER"
```

Comment:
Mind if we leave `skyux-eslint` out of the update group for now? I'm thinking we'll want a schematics package for the eslint stuff, and that package would include `skyux-eslint` in its update group, instead.